website/generated/docs/specification.md

```diff
@@ -1029,23 +1077,34 @@ The run request is sent from the client to the server to run a build target. The
 server communicates during the initialize handshake whether this method is
 supported or not.
 
+Note that an empty run request is valid. Run will be executed in the target as
```

Comment:
It's unclear what a empty run request is. Is it a run request with a null `params`?

Reply:
Oh now I see there is a related discussion: https://github.com/build-server-protocol/build-server-protocol/discussions/636